clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -878,8 +878,7 @@ raw2trace_t::get_instr_summary(uint64 modidx, uint64 modoffs, INOUT app_pc *pc,
         hashtable_add(&decode_cache, decode_pc, desc);
         ret = desc;
     } else {
-        // TODO(mtrofin): log some rendering of the instruction summary that will be
-        // returned
+        // Log some rendering of the instruction summary that will be returned. i#3129
```

Comment:
I don't understand the comment: there is no log here.  This was a "TODO" which as my prior comment said is "XXX" in the DR style (see the wiki) and I assume it is still a comment about future work, right?  Repeating what that should look like in the DR style: `XXX i#3129: Log ... .`